.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,15 @@
-## Description
-<!-- Add a description of your PR here.
-A good description should include pointers to an issue or design document, etc.
--->
-## Upgrade Notes
-Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
-* [ ] Yes (Please label as **<code>backward-incompat</code>**, and complete the section below on Release Notes)
-
-Does this PR fix a zero-downtime upgrade introduced earlier?
-* [ ] Yes (Please label this as **<code>backward-incompat</code>**, and complete the section below on Release Notes)
-
-Does this PR otherwise need attention when creating release notes? Things to consider:
+Instructions:
+1. The PR has to be tagged with at least one of the following labels:
+   1. `feature`
+   2. `bugfix`
+   3. `performance`
+   4. `ui`
+   5. `backward-incompat`
+   6. `release-notes` (*)
+2. Remove these instructions before publishing the PR.
+ 
+(*) Use `release-notes` label for scenarios like:
```

Comment:
Can we but description like these for all the labels mentioned in step 1. Most of the labels are self-explanatory but we can avoid their abuse by limiting the scope for each.

Reply:
Another thing that should be mentioned is that tests are compulsory.

Reply:
Since it’s an Apache project contributors can’t apply the labels, so it’s up to committers to do it, so “compulsory” might be off putting to first time contributors. If the author is a committer they should do it, otherwise the reviewer does it. It’s compulsory for committers, but that’s a small group of people do it’s hard not to get the memo.

Reply:
I am a little reluctant to add too much text to this at this time. Remember, we can always modify it going forward. Let us own the file and improve as we move forward.

That said, if the committers feel that more text is better, sure.

Reply:
I was only talking about mentioning that unit-tests should be there. But yeah, we can decide on that and add it later.